SPECS/systemd/systemd-bootstrap.spec

```diff
@@ -11,6 +11,10 @@ Source0:        https://github.com/systemd/systemd-stable/archive/v%{version}.ta
 Source1:        50-security-hardening.conf
 Source2:        systemd.cfg
 Source3:        99-dhcp-en.network
+# Upstream fix for journald-audit issue 
+# https://github.com/systemd/systemd/commit/df4ec48f45f518b6926e02ef4d77c8ed1a8b4e2c 
+# No longer needed when upgrading to v251+ 
```

Comment:
This info should live in the patch file directly, along with the upstream commit message from the patch.

You can get the patch incl the upstream info by putting .patch at the end of the commit URL.
https://github.com/systemd/systemd/commit/df4ec48f45f518b6926e02ef4d77c8ed1a8b4e2c.patch

You can then add a Signed-off-by from yourself to show you were the dev who included this patch.

Reply:
Ah, I added .diff to the url instead which produced the current patch. I'll swap them out and rebuild